docs/pages/demo.rst

```diff
@@ -187,7 +187,7 @@ launch the following in a new terminal and leave it running (``y`` is a convenie
     export UAVCAN__UDP__IFACE=127.0.0.1
     y sub --with-metadata uavcan.node.heartbeat uavcan.diagnostic.record    # You should see heartbeats
 
-Now let's see how the simple thermostat node is operating.
+Now let's see how the simple heater_voltage (Message) operates.
```

Comment:
Correct? As I understand it: `heater_voltage` is a Message block (I keep wanting to say Message service, but I guess that would be a bit confusing) inside the `demo_app` node?